### ISSUE TITLE
emmet the script tag only once 

### ISSUE THREAD

**Issue by TangiDing** [Microsoft/vscode#32315]
- VSCode Version: Code 1.15.0 (8b95971d8cccd3afd86b35d4a0e098c189294ff2, 2017-08-09T20:16:06.988Z)
- OS Version: Windows_NT ia32 10.0.15063
- Extensions: 

Extension|Author (truncated)|Version
---|---|---
html-snippets|abu|0.1.0
vscode-css-formatter|aes|0.0.2
jshintrc-generator|ben|1.0.0
path-intellisense|chr|1.4.2
jshint|dba|0.10.15
fileheadercomment|doi|0.0.3
jquerysnippets|don|0.0.1
vscode-html-css|ecm|0.1.7
webpack-snippets|gog|0.0.3
canvas-snippets|hol|0.3.2
vue|jcb|0.1.5
webpack|jer|2.0.0
docthis|joe|0.4.7
ftp-sync|luk|0.3.3
vscode-scss|mrm|0.6.2
open-in-browser|tec|1.1.0
vim|vsc|0.9.0
bootstrap-3-snippets|wcw|0.1.0
JavaScriptSnippets|xab|1.4.1
html-css-class-completion|Zig|1.8.0

---

Steps to Reproduce:

1. use the emmet to generate the html5 template
2. use the emmet to generate the script tag
3. when i want to use emmet to generate the script tag again, it doesn't work. and i try 
  it many times , only the first time i use emmet to generate script tag useful,
  and i found i can only use emmet to generate the script tag once among  my all files,
  my english is poor, i hope this issue helpful.

Reproduces without extensions: Yes/No

**Comment by akabrainstorm** [comment]
Possible dublicate: #32254

**Comment by ramya-rao-a** [comment]
Thanks for reporting @TangiDing 

Yes, this looks like duplicate of #32254

Let's follow up there
